src/main/treeDataProviders/issuesTree/descriptorTree/projectDependencyTreeNode.ts

```diff
@@ -19,6 +19,7 @@ export class ProjectDependencyTreeNode extends FileTreeNode {
     private _scannedCve?: Set<string> | undefined;
     // Is applicable if key in here
     private _applicableCve?: Map<string, CveApplicableDetails> | undefined;
+    private _nonapplicableCve?: Map<string, CveApplicableDetails> | undefined;
```

Comment:
```suggestion
    private _notApplicableCve?: Map<string, CveApplicableDetails> | undefined;
```
for all other places